services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_observer.py

```diff
@@ -146,9 +146,9 @@ async def observing_single_service(
         # With unhandled errors, let's generate and ID and send it to the end-user
         # so that we can trace the logs and debug the issue.
         user_error_msg = (
-            f"This service ({service_name}) unexpectedly failed."
-            " Our team has recorded the issue and is working to resolve it as quickly as possible."
-            " Thank you for your patience."
+            f"This service ({service_name}) unexpectedly failed. "
```

Comment:
```suggestion
            f"The service ({service_name}) experienced a problem."
```
or "encountered an issue."... we are trying to remove the "failed" work from our GUI